flaml/automl.py

```diff
@@ -1431,6 +1439,9 @@ def _decide_split_type(self, split_type):
             ), "groups must be specified for ranking task."
             assert split_type in ["auto", "group"]
             self._split_type = "group"
+        elif self._state.task == SUMMARIZATION:
```

Comment:
For the support of new tasks, several places need to be updated:
1. docstr about supported task types.
2. documentation: Use Cases/Task-Oriented AutoML; Examples/AutoML for NLP.
3. notebook
If you plan to add some of these in the next PR, please create some TODO tasks in project.

Reply:
What about NLG_TASKS?

Reply:
"What about NLG_TASKS?": resolved

Reply:
1, 2: resolved

3: where to create TODOs? I don't plan on notebook in this PR